### ISSUE TITLE
Kafka helm chart with SASL and authentication

### ISSUE THREAD

**Issue by ljupcovangelski** [airyhq/airy#3764]
We need to improve the Helm chart for Kafka, to include options for encryption and authentication on the serer side, so that it can be deployed in another Kubernetes cluster on the Internet then the `Airy Core` components.

**Comment by chrismatix** [comment]
@ljupcovangelski The Kafka chart is bundled with the airy chart, right? In what scenario would the Kafka chart be installed in a different cluster?

**Comment by ljupcovangelski** [comment]
@chrismatix I agree it is outside of the scope of deploying Airy (which includes the bundles Kafka chart). However, as we do have the chart independent and it is possible to install Kafka separately in another place (namespace) and connect Airy to it, I think it is good that we include this functionality in the chart.

**Comment by chrismatix** [comment]
@ljupcovangelski yes makes sense. Thanks for clarifying
